_data/projects/vscode.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: Visual Studio Code
+desc: Code editing. Redefined.
```

Comment:
@joaomoreno It will be better to have a good and explanatory description please.